src/style.css

```diff
@@ -0,0 +1,96 @@
+:root {
+  --grey: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.body {
```

Comment:
You set this rule for class `.body` but you don't have such a class. So it doesn't give any effect :)